package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alkemio/client-web",
-  "version": "0.76.0",
+  "version": "0.76.1",
```

Comment:
please revert to the "0.76.0" like in the target branch (both package and package-lock)